dev/fattest.simplicity/src/componenttest/topology/impl/LibertyClient.java

```diff
@@ -628,6 +628,34 @@ public ProgramOutput startClientWithArgs(boolean preClean, boolean cleanStart,
                 LOG.warning("The build is configured to run FAT tests with Java 2 Security enabled, but the FAT client " + getClientName() +
                             " is exempt from Java 2 Security regression testing.");
             }
+        } else {
+            // Check if "websphere.java.security" has been added to bootstrapping.properties
```

Comment:
This doesn't match the other one.  You should have a if java version >= 18.  You should be able to compare the files and bring the changes over.  I figured you would want the info() message added like in the other version.

Reply:
Good catch